app/domain/util/concurrency_limited_cache.rb

```diff
@@ -66,5 +71,15 @@ def decrease_concurrent_requests
         )
       end
     end
+
+    def cached_key(args)
```

Comment:
can you please add a description to this method? what is this cached key? is it cached key or cache key?

Reply:
Renamed to cache_key and added comment